docs/DeveloperGuide.md

```diff
@@ -1208,6 +1164,50 @@ Given below are instructions to test the app manually.
       * No optional fields: `edit /s 1 /c CS2103T`
       * Student index larger than number of students in the class: `edit /s 100 /c CS2103T /m Bob`
 
+### Marking a student present
+
+1. Marks a student present for the current class.
+
+   1. Prerequisites: View the first class called `CS2103T` with multiple students using the `view /c` command. There is a student in index 2 who has not been marked present.
```

Comment:
Just a small nitpick, is it "at index 2" instead of "in index 2"? Both sounds right but "at index 2" sounds better imo.

Reply:
Similar for line 1186 if you decide to change!

Reply:
Your suggestion sounds great! Will make the necessary changes